### PR TITLE
Fixed a bug that caused travel mode times to not be displayed

### PR DESCRIPTION
**Pull request by pobudzey**
Previously, a new path was created every single time any of the travel mode buttons was clicked.

Now, we preemptively create the necessary paths in onCreate() and store them in a hash map which allows us to:

- Display all the travel times right away
- Access the appropriate path and display it with setPathOnMapAsync whenever a travel mode button is clicked

Fixes bug #125

### CODE REVIEW

File: app/src/main/java/com/example/campusguide/DirectionsActivity.kt [hunk 1]
```diff
@@ -29,8 +27,8 @@ class DirectionsActivity : AppCompatActivity() {
     private lateinit var end: String
     private lateinit var startName: String
     private lateinit var endName: String
-    private var travelMode = "Driving"
-    private lateinit var path: PathPolyline
+    private lateinit var currentPath: PathPolyline
```
Comment (FGRCL) [inline app/src/main/java/com/example/campusguide/DirectionsActivity.kt:31]: Are all three travel modes computed once and then the radio button only swaps which one is displayed?

File: app/src/main/java/com/example/campusguide/DirectionsActivity.kt [hunk 2]
```diff
@@ -76,21 +97,21 @@ class DirectionsActivity : AppCompatActivity() {
             when (view.id) {
                 R.id.radio_driving ->
                     if (checked) {
-                        travelMode = "Driving"
-                        path = createPath(startName, endName, travelMode)
-                        setPathOnMapAsync(path)
+                        removePreviousPath()
```
Comment (FGRCL) [inline app/src/main/java/com/example/campusguide/DirectionsActivity.kt:101]: you could save couple lines if you can extract those radio button lambdas to a method.

**Review by FGRCL — APPROVED** [review]

File: app/src/main/java/com/example/campusguide/DirectionsActivity.kt (hunk 1 above)
Comment (pobudzey) [reply app/src/main/java/com/example/campusguide/DirectionsActivity.kt:31]: Yep, that's exactly how it works

File: app/src/main/java/com/example/campusguide/DirectionsActivity.kt (hunk 2 above)
Comment (pobudzey) [reply app/src/main/java/com/example/campusguide/DirectionsActivity.kt:101]: It'll save us a couple lines but make the code less understandable imo
I'll leave it as is but keep your suggestion in mind if the lambdas get more complex

**Pull request merged**